test/integration/action/action-integration.spec.js

```diff
@@ -458,24 +458,28 @@ describe('Action Integration Tests', function() {
       expect(store1.computedChannels[0].status, 'to be', 'waiting-close');
     });
 
-    it('should list no channels after mining 6 blocks', async () => {
+    it('should list closed after mining 6 blocks', async () => {
       await mineAndSync({ blocks: 6 });
-      while (store1.pendingChannels.length) await nap(100);
-      expect(store1.computedChannels.length, 'to be', 0);
+      while (!store1.closedChannels.length) {
+        await nap(100);
+        channels1.update();
```

Comment:
Shouldn't this be called inside the actions to make sure the store is updated?

Reply:
Do you mean the polling?

Reply:
You’re right. I tried call channel.update() on the channel close event and it didn’t list the closed channels yet. The channels are updated when navigating to the view anyway. Just not while the channel list is displayed. Not a big deal imho though.